gandiva_rust_udf_macro/src/udf_macro_test.rs

```diff
@@ -31,7 +31,8 @@ mod macro_tests {
 
             #[no_mangle]
             pub extern "C" fn my_udf_() -> f64 {
-                my_udf()
+                let result = my_udf();
+                result
```

Comment:
the purpose to make such changes?

Reply:
You need to check out the change for the macro. The macro now conditionally generate the code, if the `result` is `Result`, different code will be generated. To re-use most of the code template, the generated code in this case is slightly more complex.